keps/349-all-or-nothing/README.md

```diff
@@ -253,11 +271,20 @@ condition, so the corresponding job is unsuspended without further waiting.
 
 ### Timeout on reaching the PodsReady condition
 
-We introduce a timeout, defined in the `waitForPodsReady.timeoutSeconds` field, on reaching the `PodsReady` condition since the job
-is unsuspended (the time of unsuspending a job is marked by the Job's
-`job.status.startTime` field). When the timeout is exceeded, the Kueue's Job
+We introduce two timeouts defined in the `waitForPodsReady.timeoutSeconds` and `waitForPodsReady.replacementTimeoutSeconds` fields.
+
+First one tracks the time between job getting unsuspended (the time of unsuspending a job is marked by the Job's
+`job.status.startTime` field) and reaching the `PodsReady=true` condition.
+
+Second one tracks the time between changing `PodsReady` condition to `false` after the job is running and reaching the
```

Comment:
how do you plan to track this time?

Reply:
I plan to use timestamp in the `PodsReady` condition to calculate how much time passed

Reply:
If we wanted to support additional fault recovery capabilities, it could be desirable to have a separate `PodsUnhealthy` condition on the Workload instead of overloading `PodsReady`.

Reply:
What semantic differences do you imagine between `PodsUnhealthy`/`WorkloadUnhealthy` and `PodsReady`?